install_actor_deps.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+
+import sys
+import os
+from subprocess import check_call, CalledProcessError
+
+
+def error(msg, rc):
+    sys.stderr.write(msg)
+    sys.exit(rc)
+
+
+def install_deps(path):
+    cmd = "make -f {} install-deps".format(path)
+    try:
+        check_call(cmd, shell=True)
+    except CalledProcessError as e:
+        error(str(e)+'\n', e.returncode)
+
+
+def find_actor_makefile(actor):
+    for root, dirs, files in os.walk('./src/actors'):
+        if actor in dirs:
+            makefile_path = os.path.join(root, actor, 'Makefile')
+            if os.path.isfile(makefile_path):
+                return True, "Found Makefile for actor '{}'.".format(actor), makefile_path
+            return False, "Actor '{}' doesn't have Makefile!\n".format(actor), None
+    return False, "Actor '{}' doesn't exist!\n".format(actor), None
+
+
+if __name__ == "__main__":
+    try:
+        actor = sys.argv[1]
+    except IndexError:
+        error("Missing actor name!\nusage: python install_actor_deps.py ACTOR\n", 1)
+
+    passed, status, path = find_actor_makefile(actor)
+    if not passed:
+        error(status, 1)
+
+    print(status)
+    install_deps(path)
```

Comment:
I would move this file into tests/ directory, so it is not polluting the root dir. We already have schema check and test detection scripts in tests/, so it would be nice to keep everything together.

Reply:
We should probably also rename `tests` to something like `test_utils`.

Reply:
+1 If you want to go this way, please make sure it is nicely organized.